opentelemetry-example/src/main/scala/zio/telemetry/opentelemetry/example/http/StatusService.scala

```diff
@@ -1,47 +1,40 @@
 package zio.telemetry.opentelemetry.example.http
 
-import io.circe.Encoder
-import io.circe.syntax._
 import io.opentelemetry.api.trace.SpanKind
 import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator
 import io.opentelemetry.context.propagation.{ TextMapGetter, TextMapPropagator }
-import org.http4s._
-import org.http4s.circe.jsonEncoderOf
-import org.http4s.dsl.Http4sDsl
-import org.http4s.util.CaseInsensitiveString
-import zio.interop.catz._
 import zio.telemetry.opentelemetry.Tracing
 import zio.telemetry.opentelemetry.TracingSyntax._
 import zio.telemetry.opentelemetry.example.http.{ Status => ServiceStatus }
+import zhttp.http._
+import zio.json._
+import zio.ZIO
 
 import java.lang
 import scala.jdk.CollectionConverters._
 
 object StatusService {
 
-  val dsl: Http4sDsl[AppTask] = Http4sDsl[AppTask]
-  import dsl._
-
-  implicit def encoder[A: Encoder]: EntityEncoder[AppTask, A] = jsonEncoderOf[AppTask, A]
-
   val propagator: TextMapPropagator  = W3CTraceContextPropagator.getInstance()
-  val getter: TextMapGetter[Headers] = new TextMapGetter[Headers] {
-    override def keys(carrier: Headers): lang.Iterable[String] =
-      carrier.toList.map(_.name.value).asJava
+  val getter: TextMapGetter[List[Header]] = new TextMapGetter[List[Header]] {
+    override def keys(carrier: List[Header]): lang.Iterable[String] =
+      carrier.map(_.name.toString).asJava
 
-    override def get(carrier: Headers, key: String): String =
-      carrier.get(CaseInsensitiveString(key)).map(_.value).orNull
+    override def get(carrier: List[Header], key: String): String = {
+      carrier.find(_.name.toString == key).map(_.value.toString).orNull
+    }
   }
 
-  val routes: HttpRoutes[AppTask] = HttpRoutes.of[AppTask] { case request @ GET -> Root / "status" =>
-    val response = for {
-      _        <- Tracing.addEvent("event from backend before response")
-      response <- Ok(ServiceStatus.up("backend").asJson)
-      _        <- Tracing.addEvent("event from backend after response")
-    } yield response
-
-    response.spanFrom(propagator, request.headers, getter, "/status", SpanKind.SERVER)
-
-  }
+  val routes: HttpApp[Tracing, Throwable] =
+    Http.collectM {
+      case request@Method.GET -> Root / "status" =>
+      val response = for {
+        _ <- Tracing.addEvent("event from backend before response")
+        response <- ZIO.succeed(Response.jsonString(ServiceStatus.up("backend").toJson))
+        _ <- Tracing.addEvent("event from backend after response")
```

Comment:
Formatting is a bit odd here, make sure to run the `sbt fmt`. Presuming you didn't do it so far, I propose to have a separate PR for formatting, as it might cause a big diff. That one doesn't need any reviews as it's just the tool output.

Reply:
Will do.